02.wo_Defects/invalid_memory_access.c

```diff
@@ -105,6 +105,7 @@ void invalid_memory_access_003 ()
 	        c = buf;
 	        free(buf);
     	    buf = NULL; /*Tool should not detect this line as error*/ /*No ERROR:Invalid memory access to already freed area*/
+    	    c = NULL;
```

Comment:
psink  = c below will read a pointer with an indeterminate value if this is not NULLed.